variables.robot

```diff
@@ -8,212 +8,53 @@ ${MEMTEST_HEADER}=          Memtest86+ 5.01 coreboot
-...                         platform=FW6_CML    board-revision=1.01    env=dev
-...                         platform_vendor=fw66_cml    firmware_type=BIOS
-&{RTE31}=                   ip=192.168.4.121    cpuid=02c000423305c959    pcb_rev=1.1.0
-...                         platform=fw6e    board-revision=1.01    env=dev
-...                         platform_vendor=fw6e    firmware_type=BIOS
-# PCEngines developer platforms -----------------------------------------------
-&{RTE32}=                   ip=192.168.20.6    cpuid=02c000425de69477    pcb_rev=1.0.0
-...                         platform=apu1    board-revision=d4    env=dev
-...                         platform_vendor=PC Engines    firmware_type=BIOS
 # MSI-PRO-Z690-A platforms (Zir-Blazer) -----------------------------
-&{RTE33}=                   ip=192.168.10.107    cpuid=02c00042a74281e6    pcb_rev=1.1.0
-...                         platform=msi-pro-z690-a-wifi-ddr4    board-revision=1.1.0    env=dev
-...                         platform_vendor=MSI Co., Ltd    sonoff_ip=192.168.10.21
-...                         firmware_type=UEFI    pikvm_ip=192.168.10.226
-&{RTE34}=                   ip=192.168.10.199    cpuid=02c000424753a7fb    pcb_rev=1.1.0
-...                         platform=msi-pro-z690-a-wifi-ddr4    board-revision=1.1.0    env=dev
-...                         platform_vendor=MSI Co., Ltd    sonoff_ip=192.168.10.169
-...                         firmware_type=UEFI    pikvm_ip=192.168.10.153
-&{RTE39}=                   ip=192.168.10.188    cpuid=02c0014266f49b55    pcb_rev=1.1.0
-...                         platform=msi-pro-z690-a-ddr5    board-revision=1.1.0    env=dev
-...                         platform_vendor=MSI Co., Ltd    sonoff_ip=192.168.10.69
-...                         firmware_type=UEFI    pikvm_ip=192.168.10.45
-# MSI-PRO-Z790-P platforms (Zir-Blazer) -----------------------------
-&{RTE46}=                   ip=192.168.10.127    cpuid=02c00142a99e60ef    pcb_rev=1.1.0
-...                         platform=msi-pro-z790-p-ddr5    board-revision=1.1.0    env=dev
-...                         platform_vendor=MSI Co., Ltd    sonoff_ip=192.168.10.19
-...                         firmware_type=UEFI    pikvm_ip=192.168.10.226
-# PC Engines APU7 platform -----------------------------------------------------
-&{RTE35}=                   ip=192.168.10.177    cpuid=02c00042522d9294    pcb_rev=0.5.3
-...                         platform=apu7    board-revision=a    env=prod
-...                         platform_vendor=PC Engines    firmware_type=BIOS
+&{RTE33}=                   ip=192.168.10.107
+...                         platform=msi-pro-z690-a-wifi-ddr4
+...                         sonoff_ip=192.168.10.21
+&{RTE34}=                   ip=192.168.10.199
+...                         platform=msi-pro-z690-a-wifi-ddr4
+...                         sonoff_ip=192.168.10.169
+&{RTE39}=
+...                         sonoff_ip=192.168.10.69
```

Comment:
I don't understand why we leave just selected platforms. Especially that we leave only MSI DDR4 boards, but MSI DDR5 boards are removed

Reply:
@miczyg1 We need to get rid of this whole CONFIG/RTE concept, as not all stands have RTE (like this one). It made it infeasible to run any tests on laptops. With changes in this MR, we were able to.

I removed unused platforms. Z790 could stay probably.

This branch is named `novacustom-WIP` for a reason. I plan that we clean this up after releases and propose some solution to the above problem.

Reply:
Z690 DDR5 could stay too, I wish...